.github/workflows/docker-images-nightly.yml

```diff
@@ -76,7 +83,9 @@ jobs:
           push: true
           cache-from: type=gha
           cache-to: type=gha,mode=max
-          tags: ${{ secrets.DOCKER_USERNAME }}/n8n:${{ github.event.inputs.tag || 'nightly' }}
+          tags: |
+            ${{ secrets.DOCKER_USERNAME }}/n8n:${{ github.event.inputs.tag || 'nightly' }}
+            ghcr.io/${{ github.repository_owner }}/n8n:${{ github.event.inputs.tag || 'nightly' }}
```

Comment:
should we do this only for the `nightly` image? I'm not sure if it makes sense to push all our custom images to the same package on GHCR.

Reply:
I was also thinking about that. It would be nice to be able to run benchmarks for feature branches as well, which would require building a docker image out of them. Maybe we could have a separate workflow for that and push only nightly images here. WDYT?